src/scenes/Authentication/Logout/Logout.tsx

```diff
@@ -11,10 +11,14 @@ export const Logout = () => {
   const [, , setCurrentUser] = useSession();
 
   useEffect(() => {
-    logout().then(() => {
-      setCurrentUser(null);
-      navigate('/login', { replace: true });
-    });
+    logout()
+      .then(() => {
+        setCurrentUser(null);
+        navigate('/login', { replace: true });
+      })
+      .catch((e) => {
+        console.log(e);
+      });
```

Comment:
What was going wrong with logout? Were you not getting a snackbar shown with the error?

Reply:
The reason why I wrote this is because there is lint issue with this. would you recommend to add lint config for this?

Reply:
What's the lint problem?

Reply:
`14:5  error  Promises must be handled appropriately  @typescript-eslint/no-floating-promises  `

Reply:
Ah. Run `yarn` and the updated config will be applied.

Reply:
Ah, I didn't know config is already updated.